src/filesinkhelper.ipp

```diff
@@ -85,9 +85,9 @@ namespace g3 {
          return ss_entry.str();
       }
 
-      std::string createLogFileName(const std::string &verified_prefix) {
+      std::string createLogFileName(const std::string &verified_prefix, const std::string &logger_id=".g3log.") {
          std::stringstream oss_name;
-         oss_name << verified_prefix << ".g3log.";
+         oss_name << verified_prefix << logger_id;
```

Comment:
i ll give it a try,

the current branch doesnt build the tests properly on OSX
It fails with
```
[ 55%] Built target g3logger_shared
make[2]: *** No rule to make target `CMakeFiles/gtest_170_lib.dir/build'.  Stop.
make[1]: *** [CMakeFiles/gtest_170_lib.dir/all] Error 2
make: *** [all] Error 2
```